src/app/modules/users/components/users-list/users-list.component.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { MockStore, provideMockStore } from '@ngrx/store/testing';
+
+import { NgxPaginationModule } from 'ngx-pagination';
+import { UsersListComponent } from './users-list.component';
+import { UserActionTypes, UserState, LoadUsers } from '../../store';
+import { ActionsSubject } from '@ngrx/store';
+import { DataTablesModule } from 'angular-datatables';
+import { act } from '@ngrx/effects';
+
+describe('UsersListComponent', () => {
+  let component: UsersListComponent;
+  let fixture: ComponentFixture<UsersListComponent>;
+  let store: MockStore<UserState>;
+  const actionSub: ActionsSubject = new ActionsSubject();
+
+  const state = {
+    data: [{ name: 'name', email: 'email', role: 'role', id: 'id', tenand_id: 'tenand id', deleted: 'delete' }],
+    isLoading: false,
+    message: '',
+  };
+
+  beforeEach(async(() => {
```

Comment:
it is discouraged to keep using `async`, please use waitForAsync instead

Reply:
in this version it does not let me use waitForAsync, I am waiting for Sandro's changes to add it and not have conflicts in .json file